render.c

```diff
@@ -13,6 +13,7 @@
 #include "wlr-layer-shell-unstable-v1-client-protocol.h"
 #include "icon.h"
 
+#define M_PI 3.1415926535897932384
```

Comment:
Pretty sure we can get this from `<math.h>`. IIRC the magic incantation is to replace `#define _POSIX_C_SOURCE 200809L` with `#define _XOPEN_SOURCE 700`.